metadata-ingestion/src/datahub/ingestion/source/ge_profiling_config.py

```diff
@@ -84,6 +84,11 @@ class GEProfilingConfig(ConfigModel):
         description="A positive integer that specifies the maximum number of columns to profile for any table. `None` implies all columns. The cost of profiling goes up significantly as the number of columns to profile goes up.",
     )
 
+    profile_if_updated_since_days: Optional[pydantic.PositiveFloat] = Field(
+        default=1,
+        description="Profile table only if it has been updated since these many number of days . `None` implies profile all tables. Only Snowflake supports this.",
```

Comment:
Remove space between days and `.` in `days .`

Reply:
Done